yamllint/rules/quoted_strings.py

```diff
@@ -112,6 +115,28 @@
 
     - "localhost"
     - this is a string that needs to be QUOTED
+
+#. With ``quoted-strings: {quote-type: double, required: true,
+   skip-quoted-quote: false}``
```

Comment:
What do you think of this:
```suggestion
#. With ``quoted-strings: {skip-quoted-quote: false}``
```

to make the docs more easily understandable?

Reply:
I agree that now it looks a bit overloaded with pre-settings. We could easily drop `required: true` because it wouldn't affect test case logic. But without `quote-type: double` it won't complain about quotes at all.

Reply:
I've dropped `required: true` for now.

Reply:
Right!